src/verifier/validateOptions.spec.ts

```diff
@@ -47,6 +47,18 @@ describe('Verifier argument validator', () => {
         });
       });
     });
+
+    context('when given an unknown array argument', () => {
+      it('should return a Verifier object', () => {
+        expectSuccessWith({
```

Comment:
Should the user be aware that an invalid option was provided, and bail because of it?

otherwise they might feel that the argument is correctly being set, but it isn't

Reply:
I thought about adding a warning - but the existing behaviour (for unknown options that don't have an array value) was to silently ignore them.

I'm not sure if there are some unvalidated options - if we want to warn, we'll have to double check the list.

Reply:
hmmm, whilst that wouldn't be my preference, it is the way it is currently so I wouldn't want to break userspace too much. 

looking at the users provided file, the only invalid option I think was `tags` which is now `providerVersionTags` and `log` option is no longer in v10 of pact-js post moving to the rust core.

Reply:
that is probably one for another PR tho!